inc/template-functions.php

```diff
@@ -86,10 +86,11 @@ function wmf_get_header_cta_button_class() {
 /**
  * Get all the child terms for a parent organized by hierarchy
  *
- * @param int $parent_id ID to query against.
- * @return array List of organized IDs.
+ * @param int $parent_id The ID of the parent term to query against.
+ * @return array|false An associative array of child terms organized by their IDs,
+ *                     or false if no children are found for the given parent ID.
  */
-function wmf_get_role_hierarchy( $parent_id ) {
+function wmf_get_role_hierarchy( int $parent_id ) {
```

Comment:
If we're adding a type hint here we should also add the `int` hint to `wmf_get_posts_by_child_roles()`, so that we catch any type errors as early as possible.